packaging/installer/kickstart-static64.sh

```diff
@@ -15,155 +15,158 @@
 # library functions copied from packaging/installer/functions.sh
 
 setup_terminal() {
-	TPUT_RESET=""
-	TPUT_YELLOW=""
-	TPUT_WHITE=""
-	TPUT_BGRED=""
-	TPUT_BGGREEN=""
-	TPUT_BOLD=""
-	TPUT_DIM=""
-
-	# Is stderr on the terminal? If not, then fail
-	test -t 2 || return 1
-
-	if command -v tput >/dev/null 2>&1; then
-		if [ $(($(tput colors 2>/dev/null))) -ge 8 ]; then
-			# Enable colors
-			TPUT_RESET="$(tput sgr 0)"
-			TPUT_YELLOW="$(tput setaf 3)"
-			TPUT_WHITE="$(tput setaf 7)"
-			TPUT_BGRED="$(tput setab 1)"
-			TPUT_BGGREEN="$(tput setab 2)"
-			TPUT_BOLD="$(tput bold)"
-			TPUT_DIM="$(tput dim)"
-		fi
-	fi
-
-	return 0
+  TPUT_RESET=""
+  TPUT_YELLOW=""
+  TPUT_WHITE=""
+  TPUT_BGRED=""
+  TPUT_BGGREEN=""
+  TPUT_BOLD=""
+  TPUT_DIM=""
+
+  # Is stderr on the terminal? If not, then fail
+  test -t 2 || return 1
+
+  if command -v tput > /dev/null 2>&1; then
+    if [ $(($(tput colors 2> /dev/null))) -ge 8 ]; then
+      # Enable colors
+      TPUT_RESET="$(tput sgr 0)"
+      TPUT_YELLOW="$(tput setaf 3)"
+      TPUT_WHITE="$(tput setaf 7)"
+      TPUT_BGRED="$(tput setab 1)"
+      TPUT_BGGREEN="$(tput setab 2)"
+      TPUT_BOLD="$(tput bold)"
+      TPUT_DIM="$(tput dim)"
+    fi
+  fi
+
+  return 0
 }
-setup_terminal || echo >/dev/null
+setup_terminal || echo > /dev/null
 
 # -----------------------------------------------------------------------------
 fatal() {
-	printf >&2 "${TPUT_BGRED}${TPUT_WHITE}${TPUT_BOLD} ABORTED ${TPUT_RESET} ${*} \n\n"
-	exit 1
+  printf >&2 "${TPUT_BGRED}${TPUT_WHITE}${TPUT_BOLD} ABORTED ${TPUT_RESET} ${*} \n\n"
+  exit 1
 }
 
 run_ok() {
-	printf >&2 "${TPUT_BGGREEN}${TPUT_WHITE}${TPUT_BOLD} OK ${TPUT_RESET} ${*} \n\n"
+  printf >&2 "${TPUT_BGGREEN}${TPUT_WHITE}${TPUT_BOLD} OK ${TPUT_RESET} \n\n"
```

Comment:
Why is `${*}` not needed?

Reply:
Its never used. `shellcheck` picked this up.